### ISSUE TITLE
External Limb Damage Discussion

### ISSUE THREAD

**Issue by strawberrytan** [d3athrow/vgstation13#24245]
Recently merged 
"Human health now only takes into account the vital external organs. #24133"
https://github.com/vgstation-coders/vgstation13/pull/24133

This needs to be handled better than having the extremities have zero weighting on your overall health. It changes some things in bad ways both in medical autism and combat.

I was trying to make revivable husks on deadpop for fun since the captain's laser was dumped in my lap and this was the result, to my absolute horror.
![toomuch](https://user-images.githubusercontent.com/21017309/65545818-bc38f700-df48-11e9-8cc8-bcc8475747f8.png)

In discussions with other players, other quirks of this change are: explosions are far less deadly (and blowing up a welding tank on yourself won't kill you), burn damage has no over-the-top state for limbs like brute does and plasmamen are functionally immune to space carp and other simple mobs

What if external limbs had a 50% weighting on total health damage, so 100 damage to your extremities did 50 total health damage?

**Comment by Shadowmech88** [comment]
That might have more to do with burn damage in particular not really doing anything yet, as opposed to brute damage which at least causes internal and external bleeding and limb gibbing.

**Comment by MadmanMartian** [comment]
I've been thinking on this:
 - Point burn damage can blister, making a bleed similar to brute damage
 - Limb husking at a certain threshold, making the limb itself useless
 - Limb dusting, similar to limb gibbing

**Comment by AutismSpider** [comment]
past like 70 or 80 damage  limb is at risk of just turning into dust/ash

**Comment by DrSnips** [comment]
Limb ashing could be neat.

**Comment by agentofzog2** [comment]
This simply won't work without a full overhaul of how damage is dealt and how sturdy all damage areas are.

Making it so fucktons of limb damage cripples you even more is still bad because you're turning a instant  death situation into something where the victim can still fight back or at least scream for help. People always complain about trying to fight cyborgs without the borg-b-gone items and this is basically the same as that but with a potential 560 HP.

Sure this mostly applies to explosions and other stuff that doesn't target a specific area and people who forget to switch back to targeting the vitals after applying a bandaid but even the more direct methods of hurting people are affected thanks to that feature that sometimes redirects damage from one area to another.

**Comment by Kurfursten** [comment]
I really like blistering->bleeds, that's a real life condition. Did you know people who are immolated typically die of bloodloss?

No server simulates this afaik.

**Comment by SonixApache** [comment]
also, pain
all the damage should just translate into pain

**Comment by Shadowmech88** [comment]
@Kurfursten
I mentioned that in the original PR; I wrote up a big list of possible kinds of somewhat-realistic indirect damage that could be dealt to overall health from sustaining burns on the extremities.

**Comment by Shadowmech88** [comment]
@SonixApache 
Actually, very severe burns often don't hurt at all, because the nerve endings are completely destroyed by the heat.

**Comment by MadmanMartian** [comment]
One complaint I've seen first-hand is explosions not being lethal anymore.

Should they concentrate the damage to the torso?

**Comment by agentofzog2** [comment]
> 
> Should they concentrate the damage to the torso?

Might as well make everything concentrate damage on the torso then.
But then whats the point of limb damage besides punishing people who forget to switch the target area back to the vital areas?

**Comment by MadmanMartian** [comment]
It was moreso that you don't die from being punched in the arm by [x], simply because the arm breaks.

**Comment by agentofzog2** [comment]
> 
> 
> It was moreso that you don't die from being punched in the arm by [x], simply because the arm breaks.

Whats the benefit of forcing everyone to remember to switch target areas or do no damage?
You're asking for "muh realism" but also forgetting this is an abstract 2d atmos simulator.
This isn't a FPS where you have full control over where you aim at all times, this is spess and picking which area you want to hurt is decided by a hud button and RNG, this is tolerable when all you need to down someone is 100 points of damage anywhere but not when it could range from 100 to 560 points.

**Comment by Intigracy** [comment]
Moving away from stuns based combat is a good thing.

This is a good first move away from stuns based combat.

**Comment by Shadowmech88** [comment]
What if explosions, instead of just applying X amount of brute damage, had a wider array of effects? Make the explosions throw shrapnel that could cause bleeding, have the pressure waves cause mild brain and organ damage, that sort of thing.
What makes real explosions deadly?

**Comment by MadmanMartian** [comment]
Internal or external haemorrhaging

**Comment by MadmanMartian** [comment]
Shrapnel would be controlled by them being thrown away from the center of the explosion. Currently we have no embedding from range stuff active besides bullet projectiles snowflake "the bullet is caught in the wound."

**Comment by AI-Is-Rogue** [comment]
What about making explosions blow off your limbs? Doesn't seem like there are very many ways to lose your limbs that aren't a direct player-on-player confrontation. Might be fun to have more indirect ways to lose your arms and legs.

**Comment by agentofzog2** [comment]
I'll just summarize why I hate this change by boiling it down to 3 points.
As a reminder the three major sources of damage affected(broken) by this change are explosions, electric shocks and simple mobs, since they spread damage to random body parts instead of concentrating it in one area.

1. Crippled>Dead
It's a lot better to be crippled than dead. People are suggesting all these extra features for limb damage but it doesn't address the fact that no matter what it'll still be a massive nerf to those sources of damage if you can just (slowly) walk that shit off and/or still scream for help and our combat system still revolves around disabling your opponent as quickly as possible (it only takes one stun and you're fucked). 
2. We don't need more RNG
It adds an unnecessary layer of RNG to the most affected sources of damage. You already had to deal with the dice deciding which body parts are gonna be crippled, but now you have to deal with random crippling AND random damage. Shooting someone with an Egun four times is as reliable as ever but now I have to pray to RNGesus if I try to welderbomb someone or zap em with a 1niggawatt grid.
3. You remembered to switch back to targeting the chest after you bandaged yourself, right?
In the previous system there were parts of the body that made better targets than others but it didn't take that many more hits to down someone if you were aiming anywhere else, now you have to aim center mass or deal barely any  damage. All this change does is force everyone to remember to switch back to targeting the chest since it's always better to put someone into crit than make them move really slow.  You could make limbs really easy to rip off, but that would just mean everyone will target the legs instead which isn't better.

Also the two PRs robo put up so far don't fix jack shit and given what happened with the planned economy rework and the black market I doubt this health rework will ever be in an actually working state. Do we really need to keep playing with this broken system for that much longer?

tl;dr
We don't need to make space less lethal
If I wanted dicerolls to decide everything I'll go to a casino or play gatchashit
This isn't counter strike, it doesn't take more skill to pull off headshots
fuck c*ders

**Comment by SonixApache** [comment]
"this has raised my time-to-valids so it's shit"

**Comment by agentofzog2** [comment]
> 
> 
> "this has raised my time-to-valids so it's shit"

"this hasn't raised my time-to-valids so it's acceptable"
ok secmain

**Comment by AI-Is-Rogue** [comment]
Would it be feasible to make limb damage factor in a percentage to total health instead of all or nothing?

**Comment by strawberrytan** [comment]
I still think a 50% weighting from limb damage to total health would be nice, especially if the other half was pain or medical complications
